src/libraries/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -1796,6 +1796,127 @@ public enum VARKIND
         VAR_DISPATCH = 3,
     }
 }
+namespace System.Runtime.InteropServices.ObjectiveC
+{
+    [System.Runtime.Versioning.SupportedOSPlatformAttribute("macos")]
+    [System.Flags]
+    public enum RegisterInstanceFlags
+    {
+        None = 0,
+        ManagedDefinition = 1,
+    }
+
+    [System.Runtime.Versioning.SupportedOSPlatformAttribute("macos")]
+    [System.Flags]
+    public enum CreateObjectFlags
+    {
+        None = 0,
+        Unwrap = 1,
+        ManageLifetime = 2,
+    }
+
+    [System.Runtime.Versioning.SupportedOSPlatformAttribute("macos")]
+    [System.Flags]
+    public enum CreateBlockFlags
+    {
+        None = 0,
+    }
+
+    [System.Runtime.Versioning.SupportedOSPlatformAttribute("macos")]
+    [System.Flags]
+    public enum CreateDelegateFlags
+    {
+        None = 0,
+        Unwrap = 1,
+    }
+
+    [System.Runtime.Versioning.SupportedOSPlatformAttribute("macos")]
+    [System.CLSCompliantAttribute(false)]
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public struct ManagedObjectWrapperLifetime
+    {
+        public nuint Scratch;
+    }
+
+    [System.Runtime.Versioning.SupportedOSPlatformAttribute("macos")]
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    [System.CLSCompliantAttribute(false)]
+    public struct Instance
+    {
+        public IntPtr Isa;
+        public unsafe static T GetInstance<T>(Instance* instancePtr) where T : class => throw null;
+    }
+
+    [System.Runtime.Versioning.SupportedOSPlatformAttribute("macos")]
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    [System.CLSCompliantAttribute(false)]
+    public struct BlockLiteral
+    {
+        private int _dummyPrimitive;
+        public unsafe static T GetDelegate<T>(BlockLiteral* block) where T : Delegate => throw null;
+    }
+
+    [System.Runtime.Versioning.SupportedOSPlatformAttribute("macos")]
+    public abstract class ObjectiveCBase : IDisposable
+    {
+        public static readonly IntPtr InvalidInstanceValue = (IntPtr)(-1);
+
+        protected ObjectiveCBase() { }
+
+        /// <inheritdoc />
```

Comment:
Remove comment and new lines (I think the ref files usually don't have empty lines).